style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  list-style: none;
+}
+
+@media only screen and (max-width: 600px) {
+  header {
+    width: 100%;
+    left: 0;
+    top: 0;
+    background: #fff;
+  }
+
+  .navbar {
+    position: relative;
+    padding: 30px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+  }
+
+  .navbar.hamburger {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 24px;
+    top: calc(50% - 40px / 2);
+    border-radius: 8px;
+  }
+
+  .hamburger .bar {
+    justify-content: space-between;
+    display: flex;
+    flex-flow: column wrap;
+    width: 1.5rem;
+    height: 0.25rem;
+    background: #6070ff;
+    border-radius: 5px;
+    margin: 3px 0;
+  }
+
+  .navbar.logo {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    position: absolute;
+    width: 1.25rem;
+    height: 40px;
+    left: 12px;
+    top: calc(50% - 40px / 2);
+  }
+
+  .logo a {
+    width: 100px;
+    height: 20px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 1.25rem;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    text-decoration: none;
+  }
+
+  .headline {
+    max-width: 100vw;
+    min-height: 100vh;
+    background: #fff;
+    border-bottom-left-radius: 2rem;
+    margin-bottom: 10em;
+    padding: 2rem;
+    background-image: url('images/header-shapes mobile@2x.png');
+    background-size: 100vw;
+    background-repeat: no-repeat;
+  }
+
+  h1 {
+    margin-top: 5rem;
+    color: #344563;
+    font-size: 2.5rem;
+    text-align: left;
+    font-weight: 700;
+  }
+
+  p {
+    margin-top: 20px;
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+    text-align: left;
+    color: #344563;
+  }
+
+  .media-links {
+    display: flex;
+    justify-items: flex-start;
+    align-items: center;
+  }
```

Comment:
Your social Icons are appearing a little bit different from the design and it s a requirement that you stick to the design for this milestone🤗. I can see this bug didn't exist before you made the changes requested by the previous reviewer. Kindly fix this issue as learning to stick to the given design is a professional attribute😉

your page
<img width="236" alt="your socials" src="https://user-images.githubusercontent.com/64643995/212208390-3e23c747-9d39-4c13-849e-b4c125ee5967.png">

the design
<img width="208" alt="design socials" src="https://user-images.githubusercontent.com/64643995/212208475-c2fa707e-e143-43b5-9a87-d4b41abe488f.png">